psalm-baseline.xml

```diff
@@ -12,6 +12,12 @@
     <UndefinedMethod occurrences="1">
       <code>$intent</code>
     </UndefinedMethod>
+    <InvalidNullableReturnType occurrences="1">
+        <code>string|array|int|bool</code>
+    </InvalidNullableReturnType>
+    <NullableReturnStatement occurrences="1">
+        <code>$this->get_deposit_schedule_monthly_anchor()</code>
+    </NullableReturnStatement>
```

Comment:
Have added these to the psalm-baseline instead of adding `null` as a return type to `\WC_Payment_Gateway_WCPay::get_option`. 
Adding `null` causes multiple issues in downstream functions.